### PR TITLE
calc and save throughput metrics and delete multiple test suites api

### PR DESCRIPTION
**Pull request by kaylarizi**
1) new test run metrics - requestThroughput and bytesThroughput - calc, save and return to ui
2) new api - delete multiple test suites

# Checklist:
- [x] I have included a meaningful summary of my PR's content
- [x] I have linked my PR to the relevant issue
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes do not generate new compliance warnings / errors
- [x] I have tested that my fix or enhancement is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: api/src/services/test_suites_service.py [hunk 1]
```diff
@@ -36,7 +32,7 @@ def create_test_suite(data):
     current_app.database_manager.create(test_suite)
     return test_suite
 
-def create_test_run(start_time, end_time, algorithm, iterations, message_size, test_suite_id, status, status_message, client_metrics, server_metrics):
+def create_test_run(start_time, end_time, algorithm, iterations, message_size, test_suite_id, status, status_message, data_bytes, client_metrics, server_metrics):
```
Comment (yeudit) [inline api/src/services/test_suites_service.py:35]: requests_size

File: api/src/services/analyze_service.py [hunk 2]
```diff
@@ -61,10 +59,11 @@ def __run(algorithm, iterations, message_size):
     response = requests.post(current_app.configurations.curl_url + "/curl", headers=headers, json=payload, timeout=int(current_app.configurations.request_timeout))
 
     return __validate_response(response)
-        
+
 
 def __validate_response(response):
-    if response.status_code < 200 or response.status_code  > 299:
-        return Status.FAILED, json.dumps(response.json())
+    response_json = response.json()
```
Comment (yeudit) [inline api/src/services/analyze_service.py:65]: response_json -> data, body or something else

File: api/src/services/metrics_service.py [hunk 3]
```diff
@@ -1,22 +1,57 @@
...
+
+
+def create(test_run, client_metrics, server_metrics, data_bytes, start_time, end_time):
+    __save_resource_metrics(Metric.CLIENT_AVERAGE_CPU, Metric.CLIENT_AVERAGE_MEMORY, client_metrics, test_run)
+    __save_resource_metrics(Metric.SERVER_AVERAGE_CPU, Metric.SERVER_AVERAGE_MEMORY, server_metrics, test_run)
+    __save_throughput_metrics(Metric.MESSAGES_THROUGHPUT_PER_SECOND, Metric.BYTES_THROUGHPUT_PER_SECOND, start_time,
+                              end_time, data_bytes, test_run)
+
+
+def __save_resource_metrics(cpu_metric_name, memory_metric_name, metrics, test_run):
```
Comment (yeudit) [inline api/src/services/metrics_service.py:17]: __save_resources_metrics

File: api/src/services/metrics_service.py [hunk 4]
```diff
@@ -1,22 +1,57 @@
...
-    return client_data, server_data
+from flask import current_app
+from src.models.test_run_metric import TestRunMetric
+from src.enums.metric import Metric
+import pytz
+from dateutil import parser
+
+
+def create(test_run, client_metrics, server_metrics, data_bytes, start_time, end_time):
+    __save_resource_metrics(Metric.CLIENT_AVERAGE_CPU, Metric.CLIENT_AVERAGE_MEMORY, client_metrics, test_run)
```
Comment (yeudit) [inline api/src/services/metrics_service.py:11]: move client and server to save_resources method

**Review by ohadkoren — APPROVED** [review]

**Pull request merged**